### ISSUE TITLE
Add seconds to Banana start time info

### ISSUE THREAD

**Issue by ycendes** [transientskp/banana#78]
Currently if you sort images by 'start time' on Banana it only gives you info to the minute.  As AARTFAAC does 1s imaging, it would be great if we could have the second displayed as well.

**Comment by ajstewart** [comment]
Has this already been fixed in any AARTFAAC specific banana?

Otherwise it seems like with USE_L10N = True, the formatting needs to be stated manually. Changing line 56 in image_list.html to 

> <td>{{ image.taustart_ts|date:"N j, Y, g:i:s a" }}</td>

gives 

![image-list-seconds](https://cloud.githubusercontent.com/assets/3306161/8454106/b57d1a78-1ff2-11e5-97fe-29765e0f23f1.png)

Is that what is wanted here? Though perhaps Gijs knows a better way to set it globally? But I thought I'd post this solution here in case it could be used.

**Comment by gijzelaerr** [comment]
This has not been fixed in any  AARTFAAC specific banana.  This is a formatting issue and should be set in the template. As far as I know USE_L10N only affects the ordering and formatting of the date and time, but does not add seconds to the representation.

**Comment by ajstewart** [comment]
Yeah I thought this could just go in the base.py file but it seemed like USE_L10N was overwriting everything and I could only get it to change when I specified it in the place where it was being shown. 

Is there another template file that would do the trick? I'm still learning my way around how the code is set up, so I'd be interested to know.

**Comment by gijzelaerr** [comment]
What do you mean? `{{ image.taustart_ts|date:"N j, Y, g:i:s a" }}` doesn't
seem like a bad solution to me?

2015-07-06 16:49 GMT+02:00 Adam Stewart <notifications@github.com>:

> Yeah I thought this could just go in the base.py file but it seemed like
> USE_L10N was overwriting everything and I could only get it to change when
> I specified it in the place where it was being shown.
>
> Is there another template file that would do the trick? I'm still learning
> my way around how the code is set up, so I'd be interested to know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/transientskp/banana/issues/78#issuecomment-118878246>.
>



-- 
Gijs Molenaar
http://pythonic.nl

**Comment by ajstewart** [comment]
I just meant globally instead of entering that to each html page template? But maybe individually is the way.

**Comment by gijzelaerr** [comment]
hm, interesting point.

My first suggestion would be we create a template flag, and then apply that tag everywhere, so you can modify the format in a central location.

What we can also do is add an extra property function to the model the formats the field according to our preference:

http://stackoverflow.com/questions/28269520/django-custom-representation-of-a-field-in-templates
